### ISSUE TITLE
[2.7.8] Optimization needed - Too much SQL requests

### ISSUE THREAD

**Issue by BenoitPoulet** [centreon/centreon#4917]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**:  2.7.8

**Centreon Engine version**: 1.5.2

**Centreon Broker version**: 2.11.5

**OS**:  CentOS 6

**Additional environment details (AWS, VirtualBox, physical, etc.):** Physical

**Steps to reproduce the issue:**
1. Upgrade from 2.6
2. look you SQL stats
3. Cry

**Describe the results you received:**
A very very high jump in SQL request, too many for the simple UI change of Centreon 2.7
I'm see a jump from 200/s to near 800/s. 600/s more for just some dynamics fields.
IMHO, Optimization must be a priority or big install will soon have pb with Centreon.

**Describe the results you expected:**
Some more SQL request OK, but not a so big hole.

![graph_image php](https://cloud.githubusercontent.com/assets/2464712/22292766/22db38b4-e30d-11e6-9902-dbc332219333.png)

**Comment by dooko** [comment]
Do you witness any degradation of performance on your plateforme ?

**Comment by BenoitPoulet** [comment]
Yes, we can feel it when we got major outage, the system become slow more quickly than before.

**Comment by julienmathis** [comment]
Hello, 

Can you give me the database impacted ? 

Do you have changed parameters on broker configuration ?

Regards

**Comment by BenoitPoulet** [comment]
Hello,

What do you mean by "Can you give me the database impacted ?" ?
If you want the name of the database, i don't know how to identify activity per database, i only have global MySQL server stats. But if you can tell me how to do this, i will do it.

I didn't do any change in configuration after the upgrade process.
I just see that all my «Filter Category» has been reset. (not filter selected).
